mypyc/primitives/int_ops.py

```diff
@@ -28,47 +28,50 @@
             is_borrowed=True)
 
 # Convert from a float to int. We could do a bit better directly.
-func_op(
+c_function_op(
     name='builtins.int',
     arg_types=[float_rprimitive],
-    result_type=object_rprimitive,
-    error_kind=ERR_MAGIC,
-    emit=call_emit('CPyLong_FromFloat'),
-    priority=1)
+    return_type=object_rprimitive,
+    c_function_name='CPyLong_FromFloat',
+    error_kind=ERR_MAGIC)
 
 # int(string)
-func_op(
+c_function_op(
     name='builtins.int',
     arg_types=[str_rprimitive],
-    result_type=object_rprimitive,
-    error_kind=ERR_MAGIC,
-    emit=call_emit('CPyLong_FromStr'),
-    priority=1)
+    return_type=object_rprimitive,
+    c_function_name='CPyLong_FromStr',
+    error_kind=ERR_MAGIC)
 
 # int(string, base)
-func_op(
+c_function_op(
     name='builtins.int',
     arg_types=[str_rprimitive, int_rprimitive],
-    result_type=object_rprimitive,
-    error_kind=ERR_MAGIC,
-    emit=call_emit('CPyLong_FromStrWithBase'),
-    priority=1)
+    return_type=object_rprimitive,
+    c_function_name='CPyLong_FromStrWithBase',
+    error_kind=ERR_MAGIC)
 
 # str(n) on ints
-func_op(name='builtins.str',
-        arg_types=[int_rprimitive],
-        result_type=str_rprimitive,
-        error_kind=ERR_MAGIC,
-        emit=call_emit('CPyTagged_Str'),
-        priority=2)
+c_function_op(
+    name='builtins.str',
+    arg_types=[int_rprimitive],
+    return_type=str_rprimitive,
+    c_function_name='CPyTagged_Str',
+    error_kind=ERR_MAGIC,
+    var_arg_type=None,
+    steals=False,
```

Comment:
Style nit: you can leave out the `var_arg_type` and `steals` arguments, since the provided values are the same as the defaults.